package.json

```diff
@@ -25,24 +25,25 @@
   "dependencies": {
     "@digitalbazaar/did-method-key": "^2.0.0",
     "@digitalbazaar/ed25519-signature-2018": "^2.0.1",
-    "@digitalbazaar/ed25519-signature-2020": "^2.1.0",
+    "@digitalbazaar/ed25519-signature-2020": "^3.0.0",
     "@digitalbazaar/ed25519-verification-key-2018": "^3.1.1",
-    "@digitalbazaar/ed25519-verification-key-2020": "^2.1.1",
+    "@digitalbazaar/ed25519-verification-key-2020": "^3.1.0",
     "@digitalbazaar/webkms-client": "^6.0.0",
     "@digitalbazaar/zcapld": "^4.0.0",
     "assert-plus": "^1.0.0",
     "base64url-universal": "^1.1.0",
     "crypto-ld": "^6.0.0",
-    "did-veres-one": "14.0.0-beta.1",
+    "did-veres-one": "veres-one/did-veres-one#rc14.1.0-beta.0",
     "edv-client": "^9.0.0",
     "jsonld-signatures": "^9.0.2"
   },
   "peerDependencies": {
     "bedrock": "^4.1.1",
-    "bedrock-did-io": "^3.0.0",
-    "bedrock-edv-storage": "^9.0.0",
+    "bedrock-did-io": "digitalbazaar/bedrock-did-io#offline-veres",
+    "bedrock-edv-storage": "digitalbazaar/bedrock-edv-storage#offline-veres",
     "bedrock-https-agent": "^2.0.0",
     "bedrock-jsonld-document-loader": "^1.0.1",
+    "bedrock-meter-usage-reporter": "^1.0.0",
```

Comment:
Why is this being added?

Reply:
@dlongley it errored that it couldn't find that dependency.

Reply:
Yes, but if you search for it, it isn't used here. So the error must be due to something else.